CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+* avifImageGetExifOrientationFromIrotImir() and avifSetExifOrientation() were
+  added to internal.h.
+
+### Changed
+* Exif and XMP metadata is exported to PNG and JPEG files.
```

Comment:
I changed the behavior to drop the entire XMP payload and display a warning, instead of failing the export.

The alternative would be doing as libheif: ignore any extended XMP at JPEG import, to avoid any issue at JPEG export. The downside is that the whole XMP payload of [`dog_exif_extended_xmp_icc.jpg`](https://github.com/AOMediaCodec/libavif/tree/main/tests/data#file-dog_exif_extended_xmp_iccjpg) fits into a standard XMP chunk; the small enough extended XMP part would be dropped. I guess big XMP payloads are rare enough to keep the current behavior, even if different from libheif.